integration/spark/src/main/scala/org/apache/spark/sql/hive/CarbonFileMetastore.scala

```diff
@@ -267,6 +267,7 @@ class CarbonFileMetastore extends CarbonMetaStore {
       lookupRelation(tableIdentifier)(sparkSession)
     } catch {
       case _: NoSuchTableException =>
+        LOGGER.error(s"Table ${tableIdentifier.table} does not exist.")
```

Comment:
i think this can be debug log, else user can get confused with the error

Reply:
done